source/magpie/dataframe.d

```diff
@@ -50,6 +97,17 @@ public:
     +/
     auto display(bool getStr = false, int maxwidth = 0)
     {
+        import std.algorithm: map, reduce, max;
+        import std.conv: to;
+
+        auto gapCalc(T)(T[] arr)
+        {
+            static if(is(T == string))
+                return arr.map!(e => e.length).reduce!max;
```

Comment:
This static if branch is not needed as `to!string` on a string is a no-op. just use the line below.